app/internal/calendar_privacy.py

```diff
@@ -0,0 +1,21 @@
+from app.database.database import get_db
+from app.database.models import User
+# from app.internal.security.dependancies import (
```

Comment:
Please remove :)

Reply:
This is actually left on purpose anticipating koby's user system to be merged and then this code can be upgraded to be using his system

Reply:
Can't we add it after Koby's feature will be merged?

Reply:
It's like the "TODO" comments to be added only it's already ready to be added. I can add another comment here as well to make it clearer.

Reply:
Added comment